test/development/acceptance-app/ReactRefreshRequire.test.ts

```diff
@@ -270,11 +270,9 @@ describe('ReactRefreshRequire app', () => {
       'leaf.js',
       `log.push('init LeafV3'); export default {};`
     )
-    expect(await session.evaluate(() => (window as any).log)).toEqual([
-      'init LeafV3',
-      'init MiddleAV1',
-      'init MiddleBV1',
-    ])
+    expect((await session.evaluate(() => (window as any).log)).sort()).toEqual(
+      ['init LeafV3', 'init MiddleAV1', 'init MiddleBV1'].sort()
+    )
```

Comment:
This change isn't related to the PR description. It isn't clear that the order doesn't matter here. I would actually expect that the order does matter.  All the other assertions do include the order.